### ISSUE TITLE
Script to update Betterbird on linux

### ISSUE THREAD

**Issue by risaer** [Betterbird/thunderbird-patches#99]
I am quite new to betterbird and since there is no apt repository and no other update mechanism a wrote a small script to update it
https://github.com/risaer/betterbird-dirty-update

Let me know what you think about it.

**Comment by Betterbird** [comment]
Thanks for getting in touch. There are Arch and FlatPak packages which auto-update. BB also shows that it's out-of-date by popping open the About Box. However, Linux users not using Arch or FlatPak will have to update manually so your script can come in handy. What do you want to do with this GitHub issue?

**Comment by risaer** [comment]
Thanks for your quick response. You are right. An issue is not the right place but I did not know where else to put it.

After downloading I ran BB from my downloadfolder, always just starting it from terminal. When I saw the update-notification I was annoyed to find the website, find and download the archive, google for the right tar options and put everything in place, so I wrote the script.

I just wanted to share it. To make the script public and to collect some responses from "expert" users and to let search engines find it  to get some opinions feedback.

To be honest, I probably do not have time and motivation to make a fool-proof installer but I can imagine to leave TB behind and keep BB, so I guess I will put some improvements to the updater. Also depends on the feedback.

Maybe you can find some testers and if they think it is suitable you put a link on your site or directly take over the repository. If you like.

**Comment by Betterbird** [comment]
I've looked at your script again. Not a bad idea really to automate the installation. Are you aware that we already provide a desktop file at https://github.com/Betterbird/thunderbird-patches/blob/main/metadata/eu.betterbird.Betterbird.desktop?

Meanwhile I have added this text to our FAQ page:
Users who install Betterbird manuall on Linux may find [this script](https://github.com/risaer/betterbird-dirty-update) advantageous. Let us know what you think [here](https://github.com/Betterbird/thunderbird-patches/issues/99).

I've also left a comment in a German forum:
https://www.thunderbird-mail.de/forum/thread/90190-betterbird-aktuelle-version-102-5-0-bb24/?postID=506861#post506861

@dbonner, maybe you can make use of this as well.

I'll close this issue now. Maybe we'll take further action based on the feedback received.

**Comment by Betterbird** [comment]
Hi @risaer, looks like your script hasn't been maintained for a while. We've polished it and published it here now:
https://github.com/Betterbird/thunderbird-patches/blob/main/install-on-linux/install-betterbird.sh

**Comment by pepa65** [comment]
I have my own script that approaches the install a little different, it puts a versioned betterbird-VERSION in the Installation directory, and then points a symlink betterbird in that directory towards it. Makes it easier to roll back to the earlier version if needed.

My script will also work once the version goes past 115 and the `sha256-XXX.txt` file will have a different name.

I have inlined the .desktop file as I added a Thai translation to it.

https://gitlab.com/pepa65/misc/raw/master/updateBetterbird
